java/quarkus/config.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+  website: quarkus.io
+  version: 1.11
+
+build:
+  - mvn clean package -Dquarkus.package.type=uber-jar -Dquarkus.http.port=3000 -Pnative
```

Comment:
this is not duplicated ? I mean this configuration is already in `application.resources`

Reply:
Good point. For some reasons in my tests `quarkus.http.port=3000` wasn't being picked up from the conf file. Not sure why.

Reply:
Maybe @geoand knows what's going on here.

Reply:
It added to `application.properties`, it should definitely work. It however you are trying to change the port used for testing, then the property is `quarkus.http.test-port`.

Reply:
Just tried again. Have a look: https://gist.github.com/whiplash/8fd3004593c8f9440d32caafd13fe2a5

Reply:
I'll check it out

Reply:
I can't reproduce the problem. I just created a hello world application, set `quarkus.http.port=3000` in `application.properties` and when I launched the application is correctly started on port `3000`.

Reply:
I'll provide a repro. Later today. Thanks so much.

Reply:
I'll provide a repro repo later today. Thanks so much anyways!

Reply:
Have a look https://github.com/whiplash/quarkus-port-repro

Reply:
The problem is that you have named the config file `application.resources` instead of `application.properties`

Reply:
Thanks. Sorry about this.

Reply:
NP